### DISCUSSION TITLE
如何给 ArrayTable 初始化异步数据的时候添加 loading

### DISCUSSION THREAD

**Discussion by aicherish0320** [alibaba/formily#3713] [Q&A 2.x]
源码地址：https://codesandbox.io/s/xdfdrh

<img width="1012" alt="image" src="https://user-images.githubusercontent.com/99390387/218249487-fb83261a-48af-41b3-8282-21618825feb2.png">

**Comment by xiaosi-ss** [comment]
![image](https://user-images.githubusercontent.com/7309546/218681740-43db8dfc-4191-488b-a508-d0e6731b4ea9.png)
const filed = useField();
filed.setLoading(true);

不过感觉还是外面套个骨架屏方便
